Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using static System.Console;
+using System.Collections.Generic;
+
+namespace builder
+{
+    public interface IPizzaBuilder
+    {
+        void BuildDough(string doughType);
+        void BuildSauce(string sauceType);
+        void BuildTopping(string toppingType);
+        void Reset(); 
+    }
+    public class WhitePizzaBuilder : IPizzaBuilder
+    {
+        private WhitePizza _product = new();
+        public WhitePizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this._product = new WhitePizza();
+        }
+        public void BuildDough(string doughType = null )
+        {
+            if(doughType == null || string.IsNullOrEmpty(doughType) )
+            {
+                this._product.Add("White Dough");
```

Comment:
It Is a great Job that you used a default value that can help the client if they didn't know what to choose 👏👏